app/models/group.rb

```diff
@@ -1,23 +1,20 @@
 class Group < ActiveRecord::Base
+  include Joinable
```

Comment:
Nice! I like that. Very clean.

![](http://media.tumblr.com/cb84d2a1a5b426c377a7fc3bc34ac042/tumblr_inline_nbhw3gf0D31rnm0ke.gif)